src/main/java/seedu/planner/logic/parser/ParserUtil.java

```diff
@@ -56,7 +56,7 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
      * @return The module
      * @throws ParseException if the moduleCode does not meet the constraints
      */
-    private static Module parseModuleCode(String moduleCode) throws ParseException {
+    public static Module parseModuleCode(String moduleCode) throws ParseException {
```

Comment:
I think we should keep the private methods `parseModuleCode` and `parseFocusArea` private, and test them indirectly from the public methods `parseModuleCodes` and `parseFocusAreas`. Cos the former methods aren't really meant to be exposed. They're more like helper methods